app/ApiObjects/Comment.php

```diff
@@ -0,0 +1,39 @@
+<?php namespace App\ApiObjects;
+
+use App\ApiObjects\User as ApiUser;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Contracts\Support\Jsonable;
+use Illuminate\Support\Facades\Auth;
+
+class Comment implements Arrayable, Jsonable
+{
+    protected $comment;
+
+    public function __construct($comment)
+    {
+        $this->comment = $comment;
+    }
+
+    public function toJson($options = 0)
+    {
+        return json_encode($this->toArray());
+    }
+
+    public function toArray()
+    {
+        return [
+            'id' => $this->comment->id,
+            'body' => $this->comment->body,
+            'user_id' => (integer) $this->comment->user_id,
+            'user' => $this->getUser(),
+            'topic_id' => (integer) $this->comment->topic_id,
+            'created_at' => $this->comment->created_at,
+            'time_ago' => $this->comment->created_at->diffForHumans()
```

Comment:
I wonder if it wouldn't be better to use `moment.js` for something like this? I imagine you would leave it and it would say "5 seconds ago" and it will keep saying that until you refresh the SPA, which might be a bit jarring. Nice work handling it this way, though, and if you're not comfortable with integrating `moment.js` with Vue, let me know and I can take over this part.